File: Brute Force/SequentialSearch.java
class SequentialSearch
{
    public static void main(String args[])
    {
        int[] arr = {23,43,12,53,56,74,29};
        int target = 53;

        Search s1 = new Search();
        int result = s1.Seq(arr, target);

        if (result != -1)
        {
            System.out.println("Found at index: " + result);
        }
        else
        {
            System.out.println("Not found");
        }
    }
}

class Search
{
    int Seq(int[] arr, int target)
    {
        for (int i = 0; i < arr.length; i++)
        {
            if (arr[i] == target)
            {
                return i;
            }
        }
        return -1;
    }
}

File: Brute Force/SequentialSearch.cpp
#include <iostream>
#include <vector>
using namespace std;
int main()
{
	vector<int> v = {34,23,53,23,45,64};
	int target = 53;
	for (int i = 0 ; i < v.size() ; i++)
	{
		if (v[i] == target)
		{
			cout<<"The target has been found in "<<i+1;
			break;
		}
	}
}
